sys/hoon.hoon

```diff
@@ -9765,10 +9765,13 @@
     ^-  type
     ?-  -.lap
       %&  p.lap
-      %|  %-  fire
+      %|  %-  fork
           %+  turn  ~(tap in q.lap)
-          |=  {a/type b/foot}
-          [a [%dry %$ 1]]
+          |=  [a=type b=foot]
+          :-  %hold
+          ?>  ?=([%core *] a)
+          =+  dox=[%core q.q.a q.a]
+          [dox [%$ 1]]
```

Comment:
I think it could be more clear that is throwing away the foot. The sample of your gate could be `[a=type foot]`, or even `[a=type *]`. No strong feeling on this, it just took me a minute to realize that's what's happening.

Reply:
Thanks, I think this is clearer.